services/fipe/automakers.js

```diff
@@ -18,43 +18,23 @@ async function listAutomakers({ vehicleType, referenceTable }) {
     }
   );
 
-  return data
-    .map((item) => ({ nome: item.Label, valor: item.Value }))
-    .sort((a, b) => parseInt(a.valor, 10) - parseInt(b.valor, 10));
+  return sortAutomakers(data);
 }
 
-export async function listCarAutomakers(
-  { referenceTable } = { referenceTable: undefined }
-) {
-  const referenceTableCode =
-    referenceTable || (await getLatestReferenceTable());
+const sortAutomakers = (data) => data
+  .map((item) => ({ nome: item.Label, valor: item.Value }))
+  .sort((a, b) => parseInt(a.valor, 10) - parseInt(b.valor, 10));
 
+const getAutomakers = async (vehicleType, referenceTable) => {
+  if (!referenceTable) {
+    referenceTable = await getLatestReferenceTable();
+  }
```

Comment:
I'm not an expert, but I don't think it's a good practice to change the value that is received as a reference